apps/src/code-studio/initApp/loadApp.js

```diff
@@ -127,7 +127,9 @@ export function setupApp(appOptions) {
         clientState.writeSourceForLevel(appOptions.scriptName, levelId,
             +new Date(), lastSavedProgram);
       }
-      report.callback = appOptions.report.callback;
+      if (!appOptions.hasContainedLevels) {
```

Comment:
worth a comment explaining why in the hasContainedLevels case we don't need to set `report.callback`?